### PR TITLE
[Network Traffic] Convert dashboards and formatting of package

### PR DESCRIPTION
**Pull request by P1llus**
## What does this PR do?

This is part of a bigger cleanup that relates to https://github.com/elastic/security-team/issues/5910.

I tried to add as much of the filtering as a separate commit, so most of the changes is in the commit before it, making a review easier.
However the ingest pipeline changes for on_failure was added as a last commit

The tasks, when applicable was:

- [x] Formatting with https://github.com/efd6/fumpt
- [x] Add dynamic ECS template
- [x] Update format_version to 2.6.0+
- [x] Update on_failure to include proper event.kind.
- [x] Update on_failure error message to be more descriptive
- [ ] Add processor tags for on_failure message
- [x] If system/pipeline tests are missing, add them when possible.
- [x] If httpjson, does it have drop processor for empty events?
- [ ] If no dashboards, add them.
- [x] If dashboards exist, replace with lens when needed, update dashboards if applicable.

The PR looks like it is changing more than it is, because most of them are formatting changes.

Dashboard changes:
1. All table of contents.
2. Convert all to lens.
3. Add/remove/rename certain visualizations to fit the layout and usefulness better
4. Changed layout a bit.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Comment by P1llus** [comment]
One more commit incoming soon, then ready

File: packages/network_traffic/manifest.yml [hunk 1]
```diff
@@ -1,32 +1,29 @@
...
 license: basic
 description: Capture and analyze network traffic from a host with Elastic Agent.
-type: integration
 categories:
   - network
   - security
-release: ga
 conditions:
-  kibana.version: ^8.5.3
+  kibana.version: ^8.7.1
```
Comment (efd6) [inline packages/network_traffic/manifest.yml:10]: If at all possible I would like to keep this at 8.6.2 and no higher for the short term.

File: packages/network_traffic/manifest.yml [hunk 2]
```diff
@@ -1,32 +1,29 @@
-format_version: 1.0.0
 name: network_traffic
 title: Network Packet Capture
-version: "1.14.0"
+version: 1.15.0
 license: basic
```
Comment (efd6) [inline packages/network_traffic/manifest.yml:4]:
```suggestion
```

File: packages/network_traffic/data_stream/amqp/elasticsearch/ingest_pipeline/default.yml [hunk 3]
```diff
@@ -1,46 +1,55 @@
...
+  ##
+  # Set host.mac to dash separated upper case value
+  # as per ECS recommendation
+  ##
+  - gsub:
+      field: host.mac
+      ignore_missing: true
+      pattern: '[-:.]'
+      replacement: ''
+      tag: gsub mac
```
Comment (efd6) [inline packages/network_traffic/data_stream/amqp/elasticsearch/ingest_pipeline/default.yml:16]: Avoid spaces in tags?

File: packages/network_traffic/changelog.yml [hunk 4]
```diff
@@ -1,251 +1,251 @@
 # newer versions go on top
-- version: "1.14.0"
```
Comment (efd6) [inline packages/network_traffic/changelog.yml:2]: Ideally, non-semantic reformat changes this size would be in a separate PR.

**Review by efd6 — CHANGES_REQUESTED** [review]
This is a hold purely to allow discussion of the version bump.

**Comment by P1llus** [comment]
I will be reopening this in a smaller fashion, the formatting changes caused too much issues, and its going to be easier with handling the conflicts

**Pull request closed without merging**